warp/src/tesseract/mod.rs

```diff
@@ -19,12 +20,16 @@ use parking_lot::RwLock;
 
 type Result<T> = std::result::Result<T, Error>;
 
+#[cfg(target_arch = "wasm32")]
+pub mod wasm;
```

Comment:
Here instead of having a separate module, we could use `cfg_attr` to check if the target is wasm and if so, add `wasm_bindgen` respectfully and if needed we can split the impl into multiple impl with one for wasm and one for non-wasm target.

Reply:
done